packages/parser/src/parse/process.ts

```diff
@@ -101,8 +101,6 @@ export function processTokens(
   function flatten$extends(node: momoa.ObjectNode, chain: string[]) {
     const memberKeys = node.members.map((m) => m.name.type === 'String' && m.name.value).filter(Boolean) as string[];
 
-    let extended: momoa.ObjectNode | undefined;
```

Comment:
Oh good catch 🙏. I thought Biome would have caught this; this was from an earlier refactor